### PR TITLE
Implementing callInTransaction method

### PR DESCRIPTION
**Pull request by ajaypaul-ibm**
This PR contains the implementation of  callInTransaction method which is introduced in persistence 3.2

### CODE REVIEW

**Comment by ajaypaul-ibm** [comment]
#build

File: dev/com.ibm.ws.jpa.container.v32/src/com/ibm/ws/jpa/container/v32/JPAEMFactoryV32.java [hunk 1]
```diff
@@ -93,5 +93,9 @@ public PersistenceUnitTransactionType getTransactionType() {
     public void runInTransaction(Consumer<EntityManager> work) {
     	 ivFactory.runInTransaction(work);
     }
+    @Override
+    public void callInTransaction(Function<EntityManager, R> work) {
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa.container.v32/src/com/ibm/ws/jpa/container/v32/JPAEMFactoryV32.java]: Signature is not correct and doesn't compile.

File: dev/com.ibm.ws.jpa.container.v32/src/com/ibm/ws/jpa/container/v32/JPAEMFactoryV32.java [hunk 2]
```diff
@@ -93,5 +93,9 @@ public PersistenceUnitTransactionType getTransactionType() {
     public void runInTransaction(Consumer<EntityManager> work) {
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa.container.v32/src/com/ibm/ws/jpa/container/v32/JPAEMFactoryV32.java:93]: Copyright needs to include 2024

**Review by tkburroughs — CHANGES_REQUESTED** [review]

**Comment by ajaypaul-ibm** [comment]
#libby

**Comment by ajaypaul-ibm** [comment]
#build

**Comment by ajaypaul-ibm** [comment]
#libby
#build

**Comment by ajaypaul-ibm** [comment]
#libby

**Comment by ajaypaul-ibm** [comment]
#build

**Comment by ajaypaul-ibm** [comment]
#libby

**Comment by ajaypaul-ibm** [comment]
#build

**Comment by ajaypaul-ibm** [comment]
#build

**Review by tkburroughs — APPROVED** [review]

**Pull request merged**